generators/frontend/index.js

```diff
@@ -72,24 +73,47 @@ module.exports = class extends Generator {
   }
 
   writing() {
-    this.fs.copyTpl(
-      this.templatePath(`${this.props.frontend}/**/*`),
-      this.destinationPath(''),
-      this.props,
-      {},
-      {
-        globOptions: {
-          dot: true,
-          ignore: ['.DS_Store', '**/*.(jpg|jpeg|png|gif|svg)']
-        }
-      }
-    );
+    if (this.props.frontend === 'angular-demo') {
```

Comment:
Let's add React support after we've finished fixes in other PR because this involves removing the files